### PR TITLE
dinit-log: Add text color

### PR DESCRIPTION
**Pull request by TheNofis**
I just added a couple of lines so that the logging was colored :D

### CODE REVIEW

**Comment by davmac314** [comment]
Thanks for opening a PR. To accept this I would require a few changes: since the color sequences are not universal to every terminal, and since some people wouldn't want them anyway, there should be a (documented) way to turn them off (i.e. a command-line argument). Additionally I would like the escape sequences to named macros or constants rather than embed them directly in the message strings.

File: build/build.ninja [hunk 1]
```diff
@@ -0,0 +1,238 @@
+# This is the build file for project "dinit"
```
Comment (mobin-2008) [inline build/build.ninja:1]: Looks like you pushed everything in your git working dir.

File: src/includes/service-constants.h [hunk 2]
```diff
@@ -5,6 +5,12 @@
 
 #include <control-datatypes.h>
 
+// Service set log color:
+#define COLOR_GREEN "\x1b[32m" // green
+#define COLOR_RED "\x1b[33m" // red
+#define COLOR_YELLOW "\x1b[31m" // yellow
+#define COLOR_RESET "\x1b[0m" // reset
```
Comment (mobin-2008) [inline src/includes/service-constants.h:12]: "// green" comment and similar comments are unnecessary, Names are self-explanatory.

**Comment by davmac314** [comment]
Other than cleaning up the PR so that it doesn't include a bunch of built files (as Mobin pointed out):

- I notice that the same color is used for "FAILED" as "STOPPD" status. I don't think this should be the case. A service that is "stopped" successfully does not indicate an error.
- the new `--color` option looks like it is the default anyway so the option has no effect?
- new option needs documenting in the manual pages

**Comment by davmac314** [comment]
Looks like this has gone stale, so I'll close for now; feel free to re-open in future.

**Comment by christmaswreath** [comment]
hey, i'm just bumping this as i love the idea of pretty colors in init and would love to see this be a thing.

**Pull request closed without merging**